zebra-consensus/src/checkpoint/tests.rs

```diff
@@ -210,8 +210,6 @@ async fn multi_item_checkpoint_list() -> Result<(), Report> {
 }
 
 #[tokio::test]
-// Temporarily ignore this test, until the state can handle out-of-order blocks
-#[ignore]
 async fn continuous_blockchain_test() -> Result<(), Report> {
```

Comment:
this test I think is close to working but I'm not super familiar with it and I think the issues have to do with the test and not the changes made to the state service I figure @teor2345 should probably take a look at it when they start work on monday.

Reply:
I'd like to re-check this test, once the other, simpler tests are working (and covering success and failure cases).